inst/components/scss/sidebar.scss

```diff
@@ -235,90 +267,80 @@ $bslib-sidebar-column-sidebar: Min(calc(100% - var(--bslib-sidebar-icon-size)),
 }
 
 @include media-breakpoint-down(sm) {
-  .bslib-sidebar-layout, .bslib-sidebar-layout.sidebar-right {
-    --bslib-sidebar-vert-border: none;
-    --bslib-sidebar-horiz-border: #{$bslib-sidebar-border};
-    --bslib-collapse-toggle-transform: -180deg;
-    --bslib-collapse-toggle-right-transform: -180deg;
-
-    // required by sidebar init js when `sidebar(open = "desktop")`
-    &[data-bslib-sidebar-open="desktop"] {
-      --bslib-sidebar-js-init-collapsed: true;
+  .bslib-sidebar-layout {
+    // Tell sidebar init js we're on mobile for `sidebar(open = "desktop")`
+    &, &.sidebar-right {
+      &[data-bslib-sidebar-open="desktop"] {
+        --bslib-sidebar-js-init-collapsed: true;
+      }
     }
```

Comment:
```suggestion
    &[data-bslib-sidebar-open="desktop"] {
      --bslib-sidebar-js-init-collapsed: true;
    }
```

Reply:
yes, done in 020eb11e906602ab3145221c85122397a4344c77